apps/fabric-website/src/components/App/AppState.tsx

```diff
@@ -74,6 +74,11 @@ export const AppState: IAppState = {
           getComponent: cb => require.ensure([], (require) => cb(require<any>('../../pages/Styles/LocalizationPage/LocalizationPage').LocalizationPage))
         },
         {
+          title: 'Beta Theme Generator',
+          url: '#/styles/themeGenerator',
+          getComponent: cb => require.ensure([], (require) => cb(require<any>('../../pages/Styles/ThemeGeneratorToolPage/ThemeGeneratorToolPage').ThemeGeneratorToolPage))
+        },
```

Comment:
@Jahnp this is what needs to be hidden, right?

Reply:
Yep, exactly. @phkuo--this is the link you want to add `isHiddenFromMainNav: true` to.